notebook/agentchat_custom_model.ipynb

```diff
@@ -383,6 +383,7 @@
    "source": [
     "# load model here\n",
     "\n",
+    "\n",
     "config = config_list_custom[0]\n",
```

Comment:
My apologies, this and one other change were made by pre-commit checks and added inadvertently.  I'll try again to revert them and remove the two unrelated changes like this.